src/user/SignUpForm.js

```diff
@@ -0,0 +1,80 @@
+// Sign Up Form allows anyone to sign up for an account to access their personal dashboards
+// The contact form requests First Name, Last Name, Email, Password & Confirm Password.
+// The sign up form then redirects the user to their personal dashboards
+
+import React from 'react';
+import {Form, Button} from 'react-bootstrap';
+
+class SignUpForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+
+    this.submitHandler = this.submitHandler.bind(this);
+    this.handleFormChange = this.handleFormChange.bind(this);
+    this.passwordValidator = this.passwordValidator.bind(this);
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState.confirmPassword !== this.state.confirmPassword) {
```

Comment:
Let's change this into a one liner :stuck_out_tongue_winking_eye: 
```suggestion
    if (prevState.confirmPassword !== this.state.confirmPassword) this.passwordValidator();
```

Reply:
Oooh, now that's what I call clean, sleek and sexy code! haha